lib/rules/color-function-notation/index.js

```diff
@@ -93,7 +99,7 @@ const rule = (primary, secondaryOptions, context) => {
 					});
 
 					// Remove trailing 'a' from legacy function name
-					if (LEGACY_FUNCS.has(node.value.toLowerCase())) {
+					if (LEGACY_FUNCTION_NAME.test(node.value)) {
 						node.value = node.value.slice(0, -1);
```

Comment:
[suggest] Trivial, but it's possible to shorten it a bit:

```suggestion
					if (LEGACY_FUNCTION_NAME.test(value)) {
						node.value = value.slice(0, -1);
```